Use system jQuery
Index: schooltool/src/schooltool/skin/configure.zcml
===================================================================
--- schooltool.orig/src/schooltool/skin/configure.zcml
+++ schooltool/src/schooltool/skin/configure.zcml
@@ -186,7 +186,7 @@
 	    layer="schooltool.skin.ISchoolToolLayer" />
 
   <resource name="jquery.js"
-            file="resources/jquery.min.js"
+            file="/usr/share/javascript/jquery/jquery.min.js"
 	    layer="schooltool.skin.ISchoolToolLayer" />
   <resourceDirectory
             name="jstree"
